docs/configuration/settings.rst

```diff
@@ -561,9 +562,9 @@ Project information
 |                                       | <api-utilities>`. By default, this is the major version of Kinto.        |
 +---------------------------------------+--------------------------------------------------------------------------+
 | kinto.version_prefix_redirect_enab    | By default, all endpoints exposed by Kinto are prefixed by a             |
-| led  ``True``                         | :ref:`version number <versioning>`. If this flag is enabled, the server  |
-|                                       | will redirect all requests not matching the supported version to the     |
-|                                       | supported one.                                                           |
+| led  ``True``                         | :ref:`version number <api-versioning>`. If this flag is enabled, the     |
```

Comment:
Couldn't spot the refs to `api-versioning` in the diff; might be checked it actually points at something.